hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
+
+function handleInput() {
+    const val= document.getElementById('entry').value;
+    if (isNaN(val)){
+        document.getElementById('alert').textContent='You must enter a number';
+        document.getElementById('alert').style.color = 'red';
+        return;
+    }
+    else if (val < 0) {
+        document.getElementById('alert').textContent='Your number must be positive';
+        document.getElementById('alert').style.color = 'red';
+        return;
+    }
```

Comment:
I would convert `val` to `Number` to just have tighter safe typing when doing the comparison here. you can do that by just simply saying `else if (Number(val) < 0)` since you are already checking if it's a number.